gradoop-flink/src/main/java/org/gradoop/flink/algorithms/gelly/functions/LongTupleToGellyEdgeWithLongValue.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.algorithms.gelly.functions;
+
+import org.apache.flink.api.java.tuple.Tuple4;
+import org.apache.flink.graph.Edge;
+import org.apache.flink.types.NullValue;
+import org.gradoop.common.model.impl.id.GradoopId;
+
+/**
+ * Map function to create gelly edge from long tuple.
+ */
+public class LongTupleToGellyEdgeWithLongValue
+  implements ElementToGellyEdge<Tuple4<Long, GradoopId, Long, Long>, Long, NullValue> {
+
+  /**
+   * Reuse object.
+   */
+  private Edge<Long, NullValue> reuse;
+
+  /**
+   * Constructor.
```

Comment:
Avoid this kind of description. Please also check the other map functions for this.